bldsys/cmake/template/gradle/app.build.gradle.in

```diff
@@ -0,0 +1,68 @@
+plugins {
```

Comment:
What's the difference between the build.gradle and the app build.gradle?  I know if you include the app build.gradle one must change to using gradlew app:assembleDebug etc rather than simply gradlew assembleDebug.

Reply:
`build.gradle` is suppose to have generic info in it and `app/build.gradle` will have app specific info. For example you can't have an `android{}` section in `build.gradle` anymore. You will get weird errors if you do. This is why `app/build.gradle` is introduced and `android{}` section is moved there.

Reply:
Also this whole new structure is consistent with what latest Android Studio (Bumblebee) will generate if you create a new project.